contracts/core/OTCWrapper.sol

```diff
@@ -403,21 +405,22 @@ contract OTCWrapper is Initializable, OwnableUpgradeable, ReentrancyGuardUpgrade
     function _deposit(
```

Comment:
might be cleaner to just pass in the Permit struct for user signature / mm signature

Reply:
https://github.com/ribbon-finance/GammaProtocol-OTC/pull/5/commits/f817c7f35305748e9360052af3dd5f61baacdbff